predicators/ml_models.py

```diff
@@ -371,25 +375,51 @@ def _create_optimizer(self) -> optim.Optimizer:
         """Create an optimizer after the model is initialized."""
         return optim.Adam(self.parameters(), lr=self._learning_rate)
 
+    def _reset_weights(self) -> None:
+        """(Re-)initialize the network weights."""
+        self.apply(lambda m: self._weight_reset(m, self._weight_init))
+
+    @staticmethod
+    def _weight_reset(m: torch.nn.Module, weight_init: str) -> None:
+        if isinstance(m, nn.Linear):
+            if weight_init == "default":
+                m.reset_parameters()
+            elif weight_init == "normal":
+                torch.nn.init.normal_(m.weight, std=1.0)
```

Comment:
[it looks like](https://pytorch.org/docs/stable/nn.init.html#torch.nn.init.normal_) `std=1.0` is the default, so you could just remove it if you want